src/effects/native/parametriceqeffect.cpp

```diff
@@ -0,0 +1,207 @@
+#include "effects/native/parametriceqeffect.h"
+#include "util/math.h"
+
+namespace {
+    constexpr int kBandCount = 2;
+    constexpr double kDefaultCenter1 = 1000; // 1 kHz
+    constexpr double kDefaultCenter2 = 3000; // 3 kHz
+}
+
+// static
+QString ParametricEQEffect::getId() {
+    return "org.mixxx.effects.parametriceq";
+}
+
+// static
+EffectManifest ParametricEQEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Parametric Equalizer"));
+    manifest.setShortName(QObject::tr("Param EQ"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr(
+        "An 2-band parametric equalizer based on biquad filters"));
+    manifest.setEffectRampsFromDry(true);
+    manifest.setIsMasterEQ(true);
+
+    EffectManifestParameter* lfmGain = manifest.addParameter();
+    lfmGain->setId("gain1");
```

Comment:
Now that there are 2 bands with equal frequency ranges, remove references to "lfm" and "hfm" in the variable names. Something like "band1Gain" or "gain1" would work.